.github/actions/download-branding-assets/action.yml

```diff
@@ -0,0 +1,92 @@
+name: 'Download Branding Assets'
+description: 'Downloads and organizes branding assets from the branding repository'
+inputs:
+  artifact-id:
+    description: 'The artifact ID from the branding repository'
+    required: true
+  version:
+    description: 'The branding version'
+    required: true
+  commit:
+    description: 'The commit SHA from the branding repository'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Download branding artifact
+      uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+      with:
+        name: branding-assets-${{ inputs.artifact-id }}
+        path: /tmp/branding-assets
+        github-token: ${{ github.token }}
+        repository: hoverkraft-tech/branding
+        run-id: ${{ inputs.artifact-id }}
+
+    - name: Extract and organize assets
+      uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+      with:
+        script: |
+          const fs = require('fs');
+          const path = require('path');
+          const manifest = JSON.parse(process.env.BRANDING_MANIFEST);
```

Comment:
It is forbidden to use process.env.BRANDING_MANIFEST, all env vars must come from inputs. The only action that is responsible to manipulate the manifest json is the validate-manifest action